### PR TITLE
feat: fix hu-HU translations

### PR DESCRIPTION
**Pull request by peter-gy**
Tuning the already surprisingly good AI-translated strings from #203

### CODE REVIEW

**Comment by runeb** [comment]
Thank you for your thorough review, @peter-gy! I have added a question for you about "Perspective(s)", otherwise ready to merge

**Comment by peter-gy** [comment]
No worries, @runeb. Where should I see your question?

**Review by runeb — APPROVED** [review]

**Comment by runeb** [comment]
> No worries, @runeb. Where should I see your question? I can't see any review comment or feedback on my commit.

My bad @peter-gy, I had not actually submitted it! 🙈 Should be visible now

File: locales/hu-HU/src/vision.ts [hunk 1]
```diff
@@ -58,12 +58,12 @@ export default removeUndefinedLocaleResources({
    * Label for the "Perspective" dropdown in vision settings
    * @see {@link https://www.sanity.io/docs/perspectives}
    */
-  'settings.perspective-label': 'Perspective',
+  'settings.perspective-label': 'Nézet',
```
[the remark this answers is not in the document]
Comment (peter-gy) [reply locales/hu-HU/src/vision.ts:61]: Sure, this makes sense. We also say "Perspektíva" in Hungarian, so sticking with "Perspective" should be intuitive enough. I just pushed the changes.

**Comment by peter-gy** [comment]
> > No worries, @runeb. Where should I see your question? I can't see any review comment or feedback on my commit.
> 
> My bad @peter-gy, I had not actually submitted it! 🙈 Should be visible now

All good, I just addressed your comments.

**Pull request merged**